neurodsp/spectral/utils.py

```diff
@@ -61,6 +75,20 @@ def rotate_powerlaw(freqs, spectrum, delta_exponent, f_rotation=1):
     -------
     rotated_spectrum : 1d array
         Rotated spectrum.
+
+    Examples
+    --------
+    Rotate and flatten a simulated powerspectrum:
+
+    >>> from neurodsp.sim import sim_combined
+    >>> from neurodsp.spectral import compute_spectrum
+    >>> fs = 500
+    >>> n_seconds=10
+    >>> sig = sim_combined(n_seconds, fs, components={'sim_synaptic_current': {},
```

Comment:
it should use `sim_powerlaw` here. Synaptic current has a knee, and that doesn't really rotate properly.

Reply:
Fixed